README.md

```diff
@@ -176,45 +173,48 @@ qfile-agent : Fatal error: File copy: Disk quota exceeded; Last file: <...> (err
 When the installation process completes, a number of new VMs will be available on your machine, all prefixed with `sd-`.
 
 
-### Production and Staging Environments
+### Staging Environment
 
-As of March 2020, the production environment is in beta. The staging environment should not be installed for production use or on developer machines and is for use on test machines only. If you are interested in becoming involved in development, we recommend you follow the instructions for setting up the [development environment](#development-environment) instead.
+The staging environment is intended to provide an experience closer to a production environment. For example, it will alter power management settings on your laptop to prevent suspending it to disk, and make other changes that may not be desired during day-to-day development in Qubes.
```

Comment:
> The staging environment should not be installed for production use or on developer machines and is for use on test machines only

Is an important caveat that should be preserved in the new docs. Given that the nightly dom0 RPMs used in staging are built in CI, we tell devs _only_ to use the dev environment on primary workstations, with staging only suitable for test hardware. Is this stance very cautious? Yes. I'd actually recommend that we adopt the practice of manually building dom0 staging RPMs in order to unblock use of staging on developer machines, but that's a separate discussion—for now, let's re-add that warning, then we're good to go.

Reply:
There is another warning further down; I've tried to consolidate the warning language in 1fdb2cdea6de371d59f35e9cd1449d38e46f621d